axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
That probably explains why I very rarely have good ideas :) ;)

I'll look through the rest of this in a couple of hours time but expect
it's a slam dunk. I'll go ahead and open the issues for the 4 remaining
tasks you mentioned on gitter afterwards?

On Tue, 8 Sep 2015 15:19 Owen Campbell <notifications@github.com> wrote:

> In axelrod/tests/unit/test_player.py
> <https://github.com/Axelrod-Python/Axelrod/pull/293#discussion_r38930554>:
>
> > @@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
> >
> >      name = "Player"
>
> Yeah I've thought about it more during my walk
>
> No decent idea ever occurs to me whilst sat on my arse!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/293/files#r38930554>.
>